packages/server/graphql/mutations/addComment.ts

```diff
@@ -89,6 +91,44 @@ const addComment = {
       }
     }
 
+    const {entityMap} = JSON.parse(content)
```

Comment:
-1 this technically can throw, so I'd wrap it with a try-catch, maybe extract a function to publish notifications as this resolver is getting quite big?

Reply:
Fixed